packages/office-ui-fabric-react/src/components/Dialog/index.ts

```diff
@@ -1,6 +1,9 @@
 export * from './Dialog';
 export * from './Dialog.base';
 export * from './DialogContent';
+export * from './DialogContent.base';
```

Comment:
We're not going to export the base components for now, right?

Reply:
I thought that was just for wrapper components where the style is required for the component functionality. I'll double check this one in particular.

Reply:
Double checked with David and he still thinks we should expose them, but use inline styles in very specific cases like Brian's utility component.